helm/pinot/values.yaml

```diff
@@ -320,6 +368,27 @@ server:
     endpoint: "/health"
```

Comment:
I'm not very familiar with HELM, but we do have separate endpoint on server for liveness (`/health/liveness`) and readiness (`/health/readiness`) check. Should we update this file to accommodate that?